rewards-configuration.default.yml

```diff
@@ -27,29 +27,39 @@ formattingEvaluator:
     td: 1
     hr: 0
   multipliers:
-    - type: [ISSUE, ISSUER]:
+    - type: [ISSUE, ISSUER, TASK]:
+      formattingMultiplier: 1
+      wordValue: 0.1
+    - type: [ISSUE, ISSUER, COMMENT]:
       formattingMultiplier: 1
       wordValue: 0.2
-    - type: [ISSUE, ASSIGNEE]:
+    - type: [ISSUE, ASSIGNEE, COMMENT]:
```

Comment:
"Comment" implication isn't clear to me

Reply:
It is supposed to represent a comment. Would you prefer a descriptive action such as `COMMENTED`?

Reply:
Yes consider changing them to verbs 

`ISSUER`
`COMMENTED`

Reply:
Changed `COMMENT` to `COMMENTED`, added jsdoc description for each value
https://github.com/ubiquibot/conversation-rewards/blob/559c13c9a7d548519a6434e1b8393fd0d5c8f3db/src/issue-activity.ts#L21